partners/chainlink/script/config.json

```diff
@@ -12,6 +12,6 @@
     "feeType": "native",
     "remoteChains": {
         "1868": 1,
-        "1": 1868
```

Comment:
You need to have `1:42161` here as well, for the mainnet calldata generator to work